src/angular-business/chip/chip-input/chip-input.component.ts

```diff
@@ -120,13 +125,20 @@ export class ChipInputComponent extends SbbChipsMixinBase
     this._required = coerceBooleanProperty(value);
     this.stateChanges.next();
   }
+  private _required = false;
 
-  /** @docs-private */
-  @HostBinding('class.sbb-chip-input-active')
+  /**
+   * @docs-private
+   * @deprecated
+   */
   get _isActive() {
     return !this.disabled && this._focused;
   }
 
+  /**
+   * TODO: Prefix with _
```

Comment:
Wouldn't it be better to already prefix it with "_" and provide a getter to inputElement?